README.md

```diff
@@ -1,9 +1,11 @@
-# Develop chat functionality for mobile apps using Open messaging integration in Genesys Cloud and AWS services
+# Develop a chat functionality for mobile apps using Open messaging integration in Genesys Cloud, AWS services, and Ionic framework
 
-> View the full [Develop chat functionality for mobile apps using Open messaging integration in Genesys Cloud and AWS services](https://developer.mypurecloud.com/blueprints/) article on the Genesys Cloud Developer Center.
+> View the [Develop chat functionality for mobile apps using Open messaging integration in Genesys Cloud, AWS services, and Ionic framework](https://developer.mypurecloud.com/blueprints/) article on the Genesys Cloud Developer Center.
```

Comment:
Should this link directly to the blueprint itself?  It is currently going to the general blueprint page and they would then have to find this one once they got there.  Perhaps that is on purpose?  I am not sure.